sedaro/src/sedaro/results/series.py

```diff
@@ -151,7 +157,7 @@ def from_file(cls, filename):
         '''Load series from compressed JSON file.'''
         with gzip.open(filename, 'rt', encoding='UTF-8') as json_file:
             contents = json.load(json_file)
-            return cls(contents['name'], contents['time'], contents['series'])
+            return cls(contents['name'], contents['time'], contents['series'], contents['axis'])
```

Comment:
If I'm not mistaken, won't this update break backwards compatibility with files that were saved pre-DSv3?  I may be okay with intentionally breaking compatibility but we should discuss it before we make the call.  

If we decided to break it, we should consider removing these methods in place of any persistence built into the bulk downloader.  I'll need a refreshing on how that works.